node-perf-dash.k8s.io/deployment.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: node-perf-dash
-  namespace: node-perf-dash
```

Comment:
Is it not better to keep the namespace ? I prefer to avoid using the default namespace for everything deployed on aaa.

Reply:
Removed commit with this changes